tests/testfiles/testposixpath.ipynb

```diff
@@ -0,0 +1,75 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "fits_file_url = \"https://fits.gsfc.nasa.gov/samples/NICMOSn4hk12010_mos.fits\" # oda:POSIXPath\n",
+    "fits_file_path = \"test.fits\" # oda:POSIXPath"
```

Comment:
This is a bit misleading.
In the real notebook the default value of the parameter, annotated with `oda:POSIXPath` will always be the local path (assume this path exists).
It doesn't really affect your tests, just to be consistent with the intention of this feature.

Reply:
so it would something like:

```python
os.path.join(os.getcwd(), 'test.fits')
```

Reply:
I mean, for the test, only one parameter, say `fits_file_path` is enough.
It has a path as default (e.g. `"test.fits") and at the point when notebook is executed it will also be path, this is what we are achieving in this PR.

You never expect that parameter annotated as POSIXPath has URL value

Reply:
> so it would be something like:

This won't work. 
This is a general restriction on Papermill. Expressions in the "parameters" cell either don't work or produce unexpected behaviour

Reply:
ok, thanks for the clarification

so we should expect only the case of having a URL only if passed as input parameter?

Reply:
> so we should expect only the case of having a URL only if passed as input parameter?

Yes, for this annotation it will only come as a request argument to nb2service